edgelet/iotedge/src/init/import/mod.rs

```diff
@@ -0,0 +1,701 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+mod old_config;
+
+use std::path::Path;
+
+use config::Config;
+
+use edgelet_utils::YamlFileSource;
+
+const AZIOT_KEYD_HOMEDIR_PATH: &str = "/var/lib/aziot/keyd";
+const AZIOT_CERTD_HOMEDIR_PATH: &str = "/var/lib/aziot/certd";
+const AZIOT_IDENTITYD_HOMEDIR_PATH: &str = "/var/lib/aziot/identityd";
+const AZIOT_EDGED_HOMEDIR_PATH: &str = "/var/lib/aziot/edged";
+
+/// The ID used for the device ID key (symmetric or X.509 private) and the device ID cert.
+const DEVICE_ID_ID: &str = "device-id";
+
+/// The ID used for the device CA X.509 private key and the device CA cert.
+const DEVICE_CA_ID: &str = "device-ca";
+
+/// The ID used for the trust bundle certs file.
+const TRUST_BUNDLE_ID: &str = "trust-bundle";
+
+/// The default value of the edged connect and listen management URIs.
+const AZIOT_EDGED_MANAGEMENT_URI: &str = "unix:///var/lib/aziot/edged/aziot-edged.mgmt.sock";
+
+/// The default value of the edged connect and listen workload URIs.
+const AZIOT_EDGED_WORKLOAD_URI: &str = "unix:///var/lib/aziot/edged/aziot-edged.workload.sock";
```

Comment:
This is wrong. Needs to be `/var/run`, but still be `/var/lib` for CentOS.

Reply:
Fixed by copying over the existing endpoints. Rajeev confirmed that reusing the old endpoints should be fine.